### PR TITLE
FT-02f: Add user-card component that takes user object and displays

### PR DESCRIPTION
**Pull request by HasanMusaraf**
Task2 --> subtask f
f. Add `user-card` component that takes user object and displays how it is shown in the design

### CODE REVIEW

File: app/templates/components/user-card.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="userCard">
+  {{#user-avatar user=user}}
+  {{/user-avatar}}
```
Comment (Shalini-Sivakumar) [inline app/templates/components/user-card.hbs]: Use angle brackets in the hbs file.

File: app/templates/components/user-count.hbs [hunk 2]
```diff
@@ -1,10 +1,10 @@
-<h1>user count: {{ getUser.length}}</h1>
+{{!--<h1>user count: {{ getUser.length}}</h1>--}}
 
 <div class="avatar-wrapper">
 {{#each getUser as |user|}}
 
-{{#user-avatar user=user}}
-{{/user-avatar}}
+{{#user-card user=user}}
+{{/user-card}}
```
Comment (Shalini-Sivakumar) [inline app/templates/components/user-count.hbs]: Use angle brackets in the hbs file.

`<UserCard @user={{this.user}} />`

**Review by Shalini-Sivakumar — CHANGES_REQUESTED** [review]

File: app/templates/components/user-card.hbs (hunk 1 above)
Comment (HasanMusaraf) [reply app/templates/components/user-card.hbs]: I have fixed this change.

File: app/templates/components/user-count.hbs (hunk 2 above)
Comment (HasanMusaraf) [reply app/templates/components/user-count.hbs]: I have fixed this change.

File: app/templates/components/user-card.hbs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div class="userCard">
+  <UserAvatar @user={{user}} />
```
Comment (Shalini-Sivakumar) [inline app/templates/components/user-card.hbs]: Add this here

File: app/templates/components/user-count.hbs [hunk 4]
```diff
@@ -1,8 +1,6 @@
-<h1>user count: {{getUser.length}}</h1>
-
 <div class="avatar-wrapper">
   {{#each getUser as |user|}}
-    {{#user-avatar user=user}}{{/user-avatar}}
+    <UserCard @user={{user}} />
```
Comment (Shalini-Sivakumar) [inline app/templates/components/user-count.hbs:3]: Add this here

Comment (HasanMusaraf) [reply app/templates/components/user-count.hbs:3]: I have fixed this changes

File: app/templates/components/user-card.hbs (hunk 3 above)
Comment (HasanMusaraf) [reply app/templates/components/user-card.hbs]: I have fixed this changes

**Review by Shalini-Sivakumar — APPROVED** [review]

**Pull request merged**